robottelo/cli/factory.py

```diff
@@ -1691,18 +1689,21 @@ def setup_org_for_a_custom_repo(options=None):
     """
     if(
             not options or
-            not options.get('url', None)):
+            not options.get('url')):
         raise CLIFactoryError('Please provide valid custom repo URL.')
     # Create new organization and lifecycle environment if needed
-    org_id = options.get('organization-id', make_org()['id'])
-    env_id = options.get(
-        'lifecycle-environment-id',
-        make_lifecycle_environment({u'organization-id': org_id})['id']
-    )
+    if options.get('organization-id') is None:
+        org_id = make_org()['id']
+    else:
+        org_id = options['organization-id']
+    if options.get('lifecycle-environment-id') is None:
+        env_id = make_lifecycle_environment({u'organization-id': org_id})['id']
+    else:
+        env_id = options['lifecycle-environment-id']
```

Comment:
Isn't this code significantly more complex than what was present before? What's the benefit?